### PR TITLE
Contrib gtp v2

### PR DESCRIPTION
**Pull request by Osso**
This adds gtpv2 parsing and tests as a contrib module.

### CODE REVIEW

**Comment by gpotter2** [comment]
@Osso Could you add the last tests to get 100% ? See https://codecov.io/gh/secdev/scapy/pull/472/compare#diff-73636170792F636F6E747269622F6774705F76322E7079

**Comment by Osso** [comment]
Yes, you are right, it's really close. I can add a few additional tests.

File: scapy/contrib/gtp_v2.py [hunk 1]
```diff
@@ -0,0 +1,979 @@
...
+                   BitField("instance", 0, 4),
+                   gtp.TBCDByteField("MCC", "", 2),
+                   gtp.TBCDByteField("MNC", "", 1)]
+
+
+class ULI_RAI(gtp.IE_Base):
+    name = "IE Tracking Area Identity"
+    fields_desc = [  # Only type 1 is currently supported
+        gtp.TBCDByteField("MCC", "", 2),
+        # MNC: if the third digit of MCC is 0xf, then the length of
```
Comment (gpotter2) [inline scapy/contrib/gtp_v2.py]: I don't think that we need the same comment 5times...

File: scapy/contrib/gtp_v2.py [hunk 2]
```diff
@@ -0,0 +1,979 @@
...
+        # MNC is 1 byte
+        gtp.TBCDByteField("MNC", "", 1),
+        ShortField("LAC", 0),
+        ShortField("SAC", 0)]
+
+
+class ULI_SAI(gtp.IE_Base):
```
Comment (gpotter2) [inline scapy/contrib/gtp_v2.py]: Is it normal that ULI_SAI and ULI_RAI are exactly the same ? Can you combine them ?

**Comment by Osso** [comment]
How do you retrigger codecov.io?

**Comment by gpotter2** [comment]
When the build pass, Travis ci and appveyor trigger it

**Comment by gpotter2** [comment]
Apparently, The tests have failed not because of you... just empty commit

**Comment by guedou** [comment]
@osso sometimes, the Travis OS X tests fail due to network problems.

**Pull request merged**